src/in-development/main.css

```
@@ -0,0 +1,292 @@
+@font-face {
+    font-family: "GraphikLC Semibold";
+    src: url("https://static0.megafon.ru/.blocks/fonts/graphik/MegaFonGraphikLC-Semibold-Web.woff2")
+            format("woff2"),
+        url("https://static0.megafon.ru/.blocks/fonts/graphik/MegaFonGraphikLC-Semibold-Web.woff")
+            format("woff");
+    font-weight: 600;
+    font-style: normal;
+}
+
+body {
+    background: radial-gradient(
+            50% 37.09% at 100% 50%,
+            rgba(142, 81, 194, 0.15) 0%,
+            rgba(142, 81, 194, 0) 100%
+        ),
+        radial-gradient(
+            50.14% 22.46% at 0% 27.54%,
+            rgba(15, 222, 101, 0.15) 0%,
+            rgba(15, 222, 101, 0) 100%
+        ),
+        #ffffff;
+    background-repeat: no-repeat;
+    margin: 0;
+    display: grid;
+    grid-template-rows: auto 1fr;
+    grid-template-columns: 100%;
+    height: 100%;
+    min-height: 100vh;
+    font-family: "GraphikLC Semibold", Arial, "Helvetica CY", "Nimbus Sans L",
+        sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 24px;
+}
+
+.head {
+    text-align: center;
+}
+
+.head__link {
+    display: block;
+}
+
+.head__logo {
+    width: 136px;
+    height: 42px;
+}
+
+.container {
+    padding: 0 16px;
+    width: 100%;
+    box-sizing: border-box;
+    max-width: 1280px;
+    margin: 0 auto;
+    height: 100%;
+    display: flex;
+    align-items: center;
+}
+
+.interesting-dialog {
+    width: 100%;
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    padding-top: 120px;
+    background-image: url('./in-development/elements.png');
+    background-position: 80% 85%;
+    background-size: 192px 110px;
+    background-repeat: no-repeat;
+}
+
+.interesting-dialog__phrase {
+    padding: 8px 12px;
+    border: 2px solid #000000;
+    box-sizing: border-box;
+    display: inline-block;
+    position: absolute;
+    white-space: nowrap;
+    background-color: #fff;
+}
+
+.interesting-dialog__phrase--cat {
```

Везде на проектах для модификаторов используется разделитель "_", предлагаю тут не отсупать от традиции

```suggestion
.interesting-dialog__phrase_cat {
```